neptune/internal/backends/offline_backend.py

```diff
@@ -131,14 +131,5 @@ def download_data(self, project, path, destination):
 
 class NoopObject(object):
 
-    def __getattribute__(self, item):
-        return NoopObject()
-
-    def __call__(self, *args, **kwargs):
-        return NoopObject()
-
-    def __enter__(self):
-        return NoopObject()
-
-    def __exit__(self, exc_type, exc_val, exc_tb):
-        pass
```

Comment:
I tentatively removed `__call__`, `__enter__`, and `__exit__`. I understand that `NoopObject` serves as a placeholder for `Project`, and `Project` doesn't seem to implement any of those three methods. I might be mistaken or be missing something though.